app/waf/src/controllers/wafpolicy.controller.ts

```diff
@@ -12,15 +12,18 @@ import {
   getFilterSchemaFor,
   getWhereSchemaFor,
   patch,
-  put,
   del,
   requestBody,
 } from '@loopback/rest';
 import {Wafpolicy} from '../models';
 import {WafpolicyRepository} from '../repositories';
-import uuid = require('uuid');
+import {Schema, Response, CollectionResponse} from '.';
+
 const prefix = '/adcaas/v1';
 
+const createDesc = 'WAF Policy resource that need to be created';
+const updateDesc = 'WAF Policy resource properties that need to be updated';
```

Comment:
Is it possible to in-line the variables, like the L37? 
1. They are used only once. 
2. From the view concern, it should be easier to check when we go to that line.

Reply:
Done